public/jmap/map-context.d.ts

```diff
@@ -155,3 +155,11 @@ declare interface JMapContextBeforeApplyEventParams extends JMapContextEventPara
 declare interface JMapContextAfterApplyEventParams extends JMapContextEventParams  {
   // nothing else
 }
+
+declare interface JMapContextSetActiveResult {
```

Comment:
je mettrais peut-être juste "hasBeenInitialized", parce que "has" ici est utilisé dans les deux sens ("avoir quelque chose" et "avoir fait quelque chose"), mais "has initialized" il manque quelque chose: "has initialized it's state" ou encore "has been initialized".

Sinon c'est tout bon